docs/topics/modelling_considerations.md

```diff
@@ -0,0 +1,194 @@
+# Modelling considerations
+
+There are some considerations to make when designing how to model a project in the platform. This document will give you
+some hints to avoid most common mistakes (use them as hints to guide your modelling, not as strict rules).
+
+## The IoT Platform is centered in context
+
+The central piece of Telefonica's IoT Platform is the Context Broker: a component that lets you store and query information
+about context entities. But, what is a Context Entity? Context Entities are logical abstractions that represent the elements
+of your system. Some examples are:
+
+* If you are modelling a Smart City, you may have entities like: Streetlight, Garden, Bus, Street, RecycleBin...
+* Modelling a Industry you may find useful entities like: Worker, Machine, Product, and so on.
+* When designing a Smart Parking you will have entities like: Car, User, Floor...
+
+What does it mean that those objects are Context Entities (instead of classes, objects, tables, or documents). The
+difference lays in how it is treated around the platform: modifications of the CEs can be observed by subscriptions, the
+history of those changes stored, rules can be defined to modify the global Context based on CEs changes.
+
+All the given examples have several things in common:
+
+* All of them model nouns: Context Entities are subjects of interactions; they have a history, they may have subscriptions
+to their changes, etc. This behavior does not fit well with processes (and things typically modelled as verbs or actions).
+
+* All those entities are expected to have a long life in the system. Entities are not supposed to model
+transient information (e.g.: support tickets, mails, log files). This doesn't mean short-lived entities can't be modelled
+in the system, but having those kind of entities in your project will heavily impact the performance
```

Comment:
Not sure if this is just a matter of performance... I'd suggest to use what @mrutid suggested in other point: "will have several operational issues in the platform".